docs/content/commands/npm-test.md

```diff
@@ -14,7 +14,28 @@ aliases: t, tst
 
 ### Description
 
-This runs a package's "test" script, if one was provided.
+This runs a predefined command specified in the `"test"` property of
+a package's `"scripts"` object.
+
+### Example
+
+```json
+{
+  "scripts": {
+    "test": "node test.js"
+  }
+}
+```
+
+```bash
+npm start
```

Comment:
@wraithgar just a bit confused by the example since we're using `npm start`, is that a typo? Should be `npm test`

Reply:
yeah this must have been a copy/paste snafu